src/NITTA/Synthesis/Steps/Dataflow.hs

```diff
@@ -34,6 +54,8 @@ data DataflowMetrics = DataflowMetrics
     , pNotTransferableInputs :: [Float]
     -- ^number of variables, which is not transferable for affected
     -- functions.
+    , pWaveOfUse :: Float
+    -- ^number of the first wave in which one of the target variables is used
```

Comment:
Only code helps me to understand what it is.

Reply:
renamed to `pFirstWaveOfTargetUse`